src/wait-for-angular.js

```diff
@@ -36,7 +60,8 @@ export default ClientFunction(ms => {
 
         pingTimeoutId = window.setTimeout(() => {
             clearTimeouts();
-            reject(new Error('Cannot find Angular in development mode.'));
+            reject(new Error(`Cannot find information about Angular components. Make sure that the tested application was deployed in 'development' mode.
```

Comment:
@VasilyStrelyaev

Reply:
Checked by Dirk:

Cannot find information about Angular components. The tested application should be deployed in development mode. For more information, see https://angular.io/guide/deployment.